content/en/docs/setup/install/helm/index.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Customizable Install with Helm
```

Comment:
In the past, we've added `[Experimental]` to the end of the title, plus a warning at the top that this is pre-alpha and under development.
```suggestion
title: Install with Helm [Experimental]
linktitle: Install with Helm
```